### PR TITLE
Added french translations

### PR DESCRIPTION
**Pull request by gauritiwari**
Created a folder fr in resources/lang and added french translations for the given strings.

### CODE REVIEW

File: resources/lang/fr/actions.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    'load'              => 'charge',
+    'closing'           => 'fermeture',
+    'no'                => 'Non',
+    'yes'               => 'Oui',
+    'filter_by_status'  => 'filtrer par statut',
+    'filter_by_dates'   => 'filtrer par dates',
+    'filter_by_region'  => 'filtrer par région',
+    'filter_by_village' => 'filtrer par village',
+    'filter_by_groups'  => 'filtrer par groupes',
+    'filter_by_types'   => 'filtrer par type',
```
Comment (eduayme) [inline resources/lang/fr/actions.php:29]: First letter has to be uppercase in all these strings. Please look at the English version to follow the same style in uppercase letters.

File: resources/lang/fr/activity.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+return [
+    'login'                         => 'Connecté',
+    'registration'                  => 'enregistrement',
```
Comment (eduayme) [inline resources/lang/fr/activity.php:5]: First letter has to be in uppercase.

File: resources/lang/fr/activity.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    'view_home'                     => 'Affichage de la maison',
+    'view_searches'                 => 'Affichage des recherches',
+    'view_lost_people_details'      => 'Affichage des détails des personnes perdues',
+    'view_user_details'             => 'Affichage des détails de l utilisateur',
+    'view'                          => 'Visualisation',
+    'saved_new_info_about_search'   => 'Nouvelles informations enregistrées sur la recherche',
+    'clear_logs'                    => 'Effacer les journaux',
+    'time'                          => 'Temps',
+    'description'                   => 'La description',
+    'user'                          => 'l utilisateur',
```
Comment (eduayme) [inline resources/lang/fr/activity.php:15]: First letter has to be in uppercase. Also I think it should be just "Utilisateur" and not "L'utilisateur"

File: resources/lang/fr/activity.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    'method'                        => 'Méthode',
+    'route'                         => 'Route',
+    'ip_address'                    => 'Adresse IP',
+    'browser'                       => 'Navigateur',
+    'os'                            => 'OS',
+    'to_do'                         => 'Faire',
+    'in_progress'                   => 'En cours',
+    'done'                          => 'terminé',
```
Comment (eduayme) [inline resources/lang/fr/activity.php:23]: First letter has to be in uppercase.

File: resources/lang/fr/forms.php [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    'not_is_the_lost_person'                    => 'N est-ce pas la personne perdue',
+    'is_the_contact_person'                     => 'Est la personne de contact',
+    'not_is_the_contact_person'                 => 'N est pas la personne de contact',
+    'age'                                       => 'Âge',
+    'phone'                                     => 'Téléphone',
+    'address'                                   => 'Adresse',
+    'incident'                                  => 'Incident',
+    'upa'                                       => 'Vu dernièrement à',
+    'village_last_place_seen'                   => 'village dernier endroit vu ',
+    'date_last_place_seen'                      => 'date du dernier endroit vu',
```
Comment (eduayme) [inline resources/lang/fr/forms.php:30]: First letter has to be in uppercase.

File: resources/lang/fr/forms.php [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    'is_the_lost_person'                        => 'Est la personne perdue',
+    'not_is_the_lost_person'                    => 'N est-ce pas la personne perdue',
+    'is_the_contact_person'                     => 'Est la personne de contact',
+    'not_is_the_contact_person'                 => 'N est pas la personne de contact',
+    'age'                                       => 'Âge',
+    'phone'                                     => 'Téléphone',
+    'address'                                   => 'Adresse',
+    'incident'                                  => 'Incident',
+    'upa'                                       => 'Vu dernièrement à',
+    'village_last_place_seen'                   => 'village dernier endroit vu ',
```
Comment (eduayme) [inline resources/lang/fr/forms.php:29]: Empty space at the end not needed

File: resources/lang/fr/forms.php [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    'experience_with_activity'                  => 'A de l expérience avec l activité',
+    'not_experience_with_activity'              => 'N a pas d expérience avec l activité',
+    'bring_water'                               => 'A de l eau',
+    'not_bring_water'                           => 'N a pas d eau',
+    'bring_food'                                => 'A de la nourriture',
+    'not_bring_food'                            => 'N a pas de nourriture',
+    'bring_light'                               => 'A la lumière',
+    'not_bring_light'                           => 'N a pas de lumière',
+    'bring_waterproof_clothes'                  => 'A des vêtements imperméables',
+    'not_bring_waterproof_clothes'              => 'N a pas de vêtements imperméables',
```
Comment (eduayme) [inline resources/lang/fr/forms.php:53]: I think a "'" is missing between N and a

File: resources/lang/fr/forms.php [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+    'name_respond'                              => 'Répond au nom de',
+    'whatsapp_or_gps'                           => 'Whatsapp ou GPS',
+    'aspect_description'                        => 'Description physique',
+    'clothes'                                   => 'vêtements',
+    'phisic_form'                               => 'État de forme',
+    'diseases_or_injuries'                      => 'Maladie ou blessures',
+    'medication'                                => 'Médicament',
+    'limitations_or_discapacities'              => 'Limitations ou incapacités',
+    'other'                                     => 'Autre',
+    'vehicle'                                   => 'la voiture',
```
Comment (eduayme) [inline resources/lang/fr/forms.php:88]: No articles here. Just "Voiture" please

File: resources/lang/fr/forms.php [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    'utm_x'                                     => 'UTM X',
+    'utm_y'                                     => 'UTM Y',
+    'distance_last_place_seen'                  => 'Distance jusqu au dernier lieu vu',
+    'who_localizate'                            => 'Qui a fait la localisation',
+    'lost_people_state'                         => 'Statut des personnes perdues',
+    'motive_closing'                            => 'Motif de la clôture',
+    'name_respond'                              => 'Répond au nom de',
+    'whatsapp_or_gps'                           => 'Whatsapp ou GPS',
+    'aspect_description'                        => 'Description physique',
+    'clothes'                                   => 'vêtements',
```
Comment (eduayme) [inline resources/lang/fr/forms.php:82]: First letter has to be in uppercase.

File: resources/lang/fr/login.php [hunk 10]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+    'login'    => 'S identifier',
+    'email'    => 'Email',
+    'password' => 'Mot de passe',
+    'remember' => 'Souviens-toi de moi',
+    'forgot'   => 'j ai oublié mon mot de passe',
```
Comment (eduayme) [inline resources/lang/fr/login.php:8]: First letter has to be in uppercase. Missing "'".

File: resources/lang/fr/login.php [hunk 11]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+    'login'    => 'S identifier',
```
Comment (eduayme) [inline resources/lang/fr/login.php:4]: Missing "'".

File: resources/lang/fr/main.php [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+return [
+    'home'                          => 'Accueil',
+    'welcome'                       => 'Bienvenu',
+    'search'                        => 'Chercher',
+    'practice'                      => 'Entraine toi',
+    'searches'                      => 'Recherches',
+    'practices'                     => 'Les pratiques',
+    'lost_person'                   => 'Personne perdue',
+    'login'                         => 's identifier',
```
Comment (eduayme) [inline resources/lang/fr/main.php:11]: First letter has to be in uppercase

File: resources/lang/fr/main.php [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+    'logout'                        => 'Se déconnecter',
+    'register'                      => 'S inscrire',
+    'language'                      => 'Langue',
+    'catalan'                       => 'Catalan',
+    'spanish'                       => 'espagnol',
```
Comment (eduayme) [inline resources/lang/fr/main.php:16]: First letter has to be in uppercase.

File: resources/lang/fr/main.php [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+    'english'                       => 'Anglais',
+    'version'                       => 'Version',
+    'free'                          => 'gratuit',
```
Comment (eduayme) [inline resources/lang/fr/main.php:19]: First letter has to be in uppercase

File: resources/lang/fr/main.php [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+    'documentation'                 => 'Documentation',
+    'privacy_policy'                => 'Politique de confidentialité',
+    'terms_of_service'              => 'Termes et conditions',
+    'data'                          => 'Les données',
+    'ap'                            => 'Plan d action',
+    'incident'                      => 'Incident',
+    'task'                          => 'Tâche',
+    'incidents'                     => 'Incidents',
+    'version'                       => 'Version',
+    'description'                   => 'la description',
```
Comment (eduayme) [inline resources/lang/fr/main.php:30]: First letter has to be in uppercase

File: resources/lang/fr/main.php [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+    'spanish'                       => 'espagnol',
+    'english'                       => 'Anglais',
+    'version'                       => 'Version',
+    'free'                          => 'gratuit',
+    'open_source'                   => 'Open Source',
+    'documentation'                 => 'Documentation',
+    'privacy_policy'                => 'Politique de confidentialité',
+    'terms_of_service'              => 'Termes et conditions',
+    'data'                          => 'Les données',
+    'ap'                            => 'Plan d action',
```
Comment (eduayme) [inline resources/lang/fr/main.php:25]: Missing "'".

File: resources/lang/fr/main.php [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+    'incident'                      => 'Incident',
+    'task'                          => 'Tâche',
+    'incidents'                     => 'Incidents',
+    'version'                       => 'Version',
+    'description'                   => 'la description',
+    'investigation'                 => 'Enquête',
+    'no_action'                     => 'Aucun plan d action ajouté',
+    'no_incident'                   => 'Aucun incident ajouté',
+    'finish'                        => 'terminer',
```
Comment (eduayme) [inline resources/lang/fr/main.php:34]: First letter has to be in uppercase

File: resources/lang/fr/main.php [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+    'information_collected'         => 'Informations collectées',
+    'information_collected_detail'  => 'Notre site Web peut collecter des informations personnelles.',
+    'information_purpose'           => 'Finalité des informations collectées',
+    'information_purpose_detail'    => 'Notre site Web utilise les informations collectées avec votre consentement explicite pour gérer une base de données pour l enregistrement d informations liées aux recherches ou à d autres informations pertinentes connexes.',
+    'information_control'           => 'Contrôle des informations',
+    'information_control_detail'    => 'les informations personnelles collectées ne seront pas vendues, transférées ou diffusées sans le consentement de l utilisateur, sauf sur demande d un juge sur décision judiciaire.',
```
Comment (eduayme) [inline resources/lang/fr/main.php:59]: First letter has to be in uppercase

File: resources/lang/fr/messages.php [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+    'max'             => 'Ce champ doit contenir moins de 50 caractères.',
+    'max10'           => 'Ce champ doit contenir moins de 10 caractères.',
+    'email'           => 'Ce champ doit contenir une adresse e-mail',
+    'confirmed'       => 'Ce champ ne correspond pas à la confirmation.',
+    'unique'          => 'Ce champ doit être unique dans la table de données.',
+    'numeric'         => 'Ce champ doit contenir une valeur numérique valide.',
+    'gpx_files'       => 'Seuls les fichiers GPX peuvent être téléchargés.',
+    'updated'         => ' Mis à jour avec succés.',
+    'added'           => ' ajouté avec succès.',
+    'deleted'         => ' Supprimé avec succès.',
```
Comment (eduayme) [inline resources/lang/fr/messages.php:17]: First letter has to be in lowercase.

File: resources/lang/fr/permissions_roles.php [hunk 20]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+return [
+    'permissions'    => 'Autorisations',
+    'roles'          => 'Rôles',
+    'permission'     => 'Autorisation',
+    'role'           => 'Rôle',
+    'list'           => 'liste',
```
Comment (eduayme) [inline resources/lang/fr/permissions_roles.php:8]: First letter has to be in uppercase.

**Review by eduayme — CHANGES_REQUESTED** [review]
Some uppercase letters and other problems. Please review all feedback, change it and then I will be able to merge it into the project.

File: resources/lang/fr/messages.php (hunk 19 above)
Comment (gauritiwari) [reply resources/lang/fr/messages.php:17]: lower or upper? i'll have to change all the first letters to uppercase. right?

File: resources/lang/fr/forms.php (hunk 7 above)
Comment (gauritiwari) [reply resources/lang/fr/forms.php]: Resoled the issues

Comment (gauritiwari) [reply resources/lang/fr/forms.php]: Resolved

File: resources/lang/fr/forms.php (hunk 8 above)
Comment (gauritiwari) [reply resources/lang/fr/forms.php]: done

**Comment by eduayme** [comment]
I am sorry @gauritiwari but @besnardf have already done it with no problems. I have already merge his French translation into the project.

**Pull request closed without merging**